### DISCUSSION TITLE
Welcome to codeapp Discussions!

### DISCUSSION THREAD

**Discussion by bummoblizard** [thebaselab/codeapp#286] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Mister-Hope** [comment]
说几句。

我对这款软件很失望，如果这是一个¥1或¥6的软件，那我可以理解为用爱发电，收一个开发者的费用。但既然¥30的收费，我觉得我应该有更高的期待。但是我在软件内还是这个仓库，都没有找到一个"基本"的文档。

我上手的时候就遇到了 Git 问题，按照你们的设计，git的全局配置应该只能通过设置配置一下名称和邮箱（我试过了 `git config --global` 去设置，有权限问题），但是目前我配置是无效的。在我尝试过卸载重装以及重置设置等操作后，我不觉得锅在我这里。

另外我目前为止，是没有看到任何的下载进度的。（不清楚是不是因为我的 `git clone` 压根就不成功）

还有，我在试着放弃git，先在本地体验玩具项目的时候，我发现贵软件的菜单连移动的选项都没有。我以为能支持一个基本的拖拽，或者让我通过菜单及弹窗进行重命名，而不是只能敲mv命令。另外我同样不是很清楚，为什么无论我是否选择了一个文件夹，我新建文件总是在根目录下，会以愚蠢的 default.js default(2).js 命名并注入包含谜之注释的预设内容。如果说开发者只能在终端中敲一些命令才能获得一个编辑器最基本的新建、移动、复制（包括文件和文件夹）操作的话，那我很难说这个软件完成了最基本的功能。

从一个正常的开发者的角度来说，我并没指望移动版本的node能跑起来我的什么项目，但我觉得有理由期待，一款在简介里声称"桌面编辑器"的App，能够提供一个编辑器最基本的功能（文件创建修改移动编辑，以及代码的拉取和推送），以及常见语言的高亮和关键词提示与补全。实在说，后者我觉得的确还可以，但前者我实在无话可说。

最后，请问开发者是否仍有开发计划以及定期地发布计划？我看到近两个月在销量剧增之后，开发者似乎直接把项目扔在这里了，是因为"busy in real life"还是直接弃坑了？另外能否建立可预期的版本发布？上次版本发布以来几乎没修复什么问题（issue区似乎堆了不少了），我觉得使用者应该期待自己提出的问题能够至少在1个月能够解决，否则此编辑器跟玩具有何异？怎么期待它能作为临时的替代品。

祝好

**Reply by bummoblizard** [reply]
首先感谢你的回馈。

Git方面要透过GUI操作，命令栏内的git并不完整，这个确实没有说清楚。你要到设定页面来设置帐号密码等资料，然后再版本控制一栏中输入repo url以下载。

然后重新命名方面，长按文件即可，亦可透此在根目录以外新增文件/目录。新增文件方面如果不想有预设内容，可在下方输入自定义文件名称。

最后就两个月没有太多更新我确实要道歉，会尽快开始着手处理现有的问题。

**Reply by Mister-Hope** [reply]
emm，我的意思是我在特定文件夹下新建制定文件，而且不希望里面添加了预设内容的话，要怎么操作呢？

另外文件、文件夹的移动和复制是否没有被支持。我比较期待类似vscode的方式，可以直接拖拽，或长按选择复制，选择某个文件夹粘贴进去。这样才是比较有生产力的方式。

**Reply by bummoblizard** [reply]
https://user-images.githubusercontent.com/38398443/144636032-50fe7aa5-d246-42db-aaa1-d80a58c32445.MP4

**Reply by Mister-Hope** [reply]
感谢回复，那这个我想提一个feature request，就是允许用户选定特定文件夹，然后把底下的新建文件按钮和新建文件夹按钮绑定到该文件夹下？

我觉得在根目录创建文件显然不会是一个很常见的需求....如果真的像我想象的一样，把它作为中小型项目临时的编辑器和修改器的话。

**Comment by 1-yan** [comment]
下载的第一天，感觉总体还是很不错的，可是有两个小bug希望作者大大修改下！
- 无法进行全部代码的选中
- 代码侧边同一括号内容的收起功能容易瞬间两次重复操作![image](https://user-images.githubusercontent.com/104848108/204033506-1648aa60-5bdc-4e4b-9bbd-cff0a25675d9.jpeg)

**Comment by AllFresh** [comment]
我有一个问题，就是我写好了C或者C++的程序，但是编译的时候没有结果出来。我不知道是不是少了哪一个步骤，想请求帮助，谢谢。
